src/BenchmarkDotNet/Portability/Cpu/ProcCpuInfoParser.cs

```diff
@@ -13,7 +16,12 @@ internal static CpuInfo ParseOutput([CanBeNull] string content)
             var logicalCores = SectionsHelper.ParseSections(content, ':');
             var processorModelNames = new HashSet<string>();
             var processorsToPhysicalCoreCount = new Dictionary<string, int>();
+            
             var logicalCoreCount = 0;
+            var nominalFrequency = (Frequency)0d;
+            var minFrequency = (Frequency)0d;
+            var maxFrequency = (Frequency)0d;
```

Comment:
The statement looks ugly to me. Let's introduce
```cs
public static readonly Frequency Zero = new Frequency(0);
```
in `Frequency.cs` and use `Frequency.Zero` here.